orttraining/orttraining/test/python/orttraining_test_ortmodule_deepspeed_pipeline_parallel.py

```diff
@@ -0,0 +1,119 @@
+                        default=2,
+                        help='pipeline parallelism')
+    parser.add_argument('--backend',
+                        type=str,
+                        default='nccl',
+                        help='distributed backend')
+    parser.add_argument('--seed', type=int, default=0, help='PRNG seed')
+    parser.add_argument('--fp16',type=bool,default=False,help='fp16 run')
+    parser.add_argument('--run_without_ort',type=bool,default=False,help='onlydeepspeed run')
+    
+    parser = deepspeed.add_config_arguments(parser)
+    args = parser.parse_args()
+    return args
+
+print('Running deepspeed pipeline parallel module with ORTModule')
+n = 10
+d_in = 4
+d_hidden = 8
+d_out = 3
+import sys
+print(sys.version)
+args = get_args()
+torch.cuda.set_device(args.local_rank)
+device = torch.device("cuda", args.local_rank)
+
+dist.init_process_group(backend=args.backend)
+torch.manual_seed(args.seed)
+# Model.
+
+if args.run_without_ort:
+    model = nn.Sequential(
+        nn.Linear(d_in, d_hidden),     # Stage 1
+        nn.ReLU(),                     # Stage 1
+        nn.Linear(d_hidden, d_hidden), # Stage 1
+        nn.ReLU(),                     # Stage 1
+        nn.Linear(d_hidden, d_hidden), # Stage 2
+        nn.ReLU(),                     # Stage 2
+        nn.Linear(d_hidden, d_out)     # Stage 2
+    )
+
+
+else:
+    print("*************************************** testing with ORTModule ************")
+    model = nn.Sequential(
+        ORTModule(nn.Linear(d_in, d_hidden).to(device)),     # Stage 1
+        nn.ReLU().to(device),#ORTModule(nn.ReLU().to(device)),                     # Stage 1
+        ORTModule(nn.Linear(d_hidden, d_hidden).to(device)), # Stage 1
+        nn.ReLU().to(device),#ORTModule(nn.ReLU().to(device)),                     # Stage 1
+        ORTModule(nn.Linear(d_hidden, d_hidden).to(device)), # Stage 2
+        nn.ReLU().to(device),#ORTModule(nn.ReLU().to(device)),                     # Stage 2
+        ORTModule(nn.Linear(d_hidden, d_out).to(device))     # Stage 2
+    )
+
+model = PipelineModule(layers=model,
+            loss_fn=torch.nn.CrossEntropyLoss(),
+            num_stages=args.pipeline_parallel_size,
+            partition_method='uniform', #'parameters',
+            activation_checkpoint_interval=0
+            )
+
+params = [p for p in model.parameters() if p.requires_grad]
+
+# Input.
+x = torch.rand((n, d_in))
+if args.fp16:
+    x = x.half()
+# Output.
+y = torch.randint(0, d_out, (n,))
+ds = SampleData(x,y)
+
+print("Initialize deepspeed")
+model_engine, optimizer, _, _ = deepspeed.initialize(args=args,
+                                                     model=model,
+                                                     model_parameters=params,
+                                                     training_data=ds #(x,y)#
+                                                    )
+
+for step in range(args.steps):
+    loss = model_engine.train_batch()
```

Comment:
Does `model_engine.train_batch()` print loss or any other metric

Reply:
it prints the final loss. But the latest commit  prints loss every 10 steps.